### DISCUSSION TITLE
Interest Accrual End Date

### DISCUSSION THREAD

**Discussion by magonjr** [Open-Cap-Table-Coalition/Open-Cap-Format-OCF#234] [Validation]
We propose adding a field called Interest Accrual End Date to denote when a convertible note ceases accruing interest since, occasionally, the date the interest stops accruing is not the same as the conversion date. We only saw a conversion date field in OCF, so wasn’t sure if such an edge case could be handled.

**Comment by JSv4** [comment]
Opened an issue for this in #249
